Rewrite in-app documentation (#3599)
gchq/Bailo: frontend/pages/docs/administration/federation/peer-integration.mdx

## Peer Types

Bailo supports connecting to different types of external model sources.

- **Bailo** - Another Bailo instance
- **HuggingFace Hub** - The HuggingFace model repository

Additional sources can be set up by your Bailo administrator.

## Configuring Peers

Peers are configured in the application's backend configuration. Each peer requires:

- **Base URL**: The URL of the external Bailo instance or repository
- **Label**: A display name for the peer (shown in the Marketplace filter)
- **Kind**: The type of peer (Bailo or HuggingFace Hub)
- **State**: Whether federation with this peer is disabled, read-only, or enabled

Peer configuration is done through the app configuration file - see
[App Configuration](/docs/administration/getting-started/app-configuration).

## How Federation Appears to Users

Users see external models alongside local models in the Marketplace, with filtering by external source.

When federation is enabled and peers are configured:

- Users see a **filter by external sources** option in the Marketplace
- Models from external sources appear alongside local models
- Each external model shows its source

> - External models can be mirrored to create local read-only copies

Reviewer: ```suggestion
- External models can be clicked to view the source
```

## Checking Peer Status

Check the reachability of connected peers through the API.

You can check the status of connected peers [via the API](/api/v2/docs/#/system).

Each peer has a reachability status indicating whether Bailo can communicate with it.

## Model Mirroring via Federation

Users can create read-only mirrored models from federated external sources.

When federation is enabled, users can create **Mirrored Models** from external sources. These are read-only copies that
require periodic updates to stay in sync with the source. See

> [Creating a Mirrored Model](/docs/users/model-mirroring/creating-a-mirrored-model) for details.

Reviewer: ```suggestion
```
Not a thing currently
